server/main.go

```diff
@@ -78,6 +83,25 @@ func main() {
 	}
 	defer db.Close()
 
+	dbconf, err := goose.NewDBConf(*dbConfDir, *env, "")
```

Comment:
How about putting all of this into a function? Something like `ensureDBMigrated`?